QUALITY_DECLARATION.md

```diff
@@ -1,20 +1,22 @@
-This document is a declaration of software quality for the `rcutils` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+This document is a declaration of software quality for the `rcutils` package, based on the guidelines in [REP-2004](https://github.com/ros-infrastructure/rep/blob/rep-2004/rep-2004.rst).
 
 # `rcutils` Quality Declaration
 
-The package `rcutils` claims to be in the **Quality Level 1** category.
+The package `rcutils` claims to be in the **Quality Level 4** category.
 
-Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 1 in REP-2004](https://www.ros.org/reps/rep-2004.html).
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 1 in REP-2004 ](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#quality-level-1) of the ROS2 developer guide.
 
 ## Version Policy
```

Comment:
In the REP 2004 there is a guideline for Quality Declaration in `markdown` which add the index referencing the REP to the title. Right now looks like:

```
## Version Policy
```
But the guideline define this as
```
## Version Policy [1]
```

I suggest to change all the titles to follow the section number as defined in the REP